### ISSUE TITLE
Is this repo no longer being maintained?

### ISSUE THREAD

**Issue by skywalker023** [abhshkdz/ai-deadlines#723]
Hi,
It seems this repository hasn’t been updated in over two months. Is it no longer being maintained?
I like this site and I would love to see it continue. It would be great if we have someone who can actively merge the pull requests.
Thank you!

**Comment by skywalker023** [comment]
@rstojnic @abhshkdz @andrewkuanop
Tagging people who seem to have the access to merge!

**Comment by HireTheHero** [comment]
For those who are wondering if you should DIY, this repo seems to require some manual effort for the conference update (see [conferences.yml](https://github.com/paperswithcode/ai-deadlines/blob/gh-pages/_data/conferences.yml) and the previous PRs such as #704).

**Comment by skywalker023** [comment]
@rstojnic @abhshkdz @andrewkuanop 
Tagging once again in case you missed it :)

**Comment by abhshkdz** [comment]
Sorry about this. Sad to see the state of affairs here :( 

I'm no longer at Meta (and neither is @rstojnic I believe?), and don't have merge permissions and/or the bandwidth to maintain this unfortunately.

Bandwidth was the reason I'd handed over ownership to paperswithcode in the first place. I'll try pinging some folks there..

**Comment by heitorrapela** [comment]
Thanks for open the issue @skywalker023 and thanks for the effort @abhshkdz! 

I hope you guys manage to keep this repo working, it was my go to place to check the conferences! :)

**Comment by joachimneu** [comment]
Hey, I am maintaining a fork of this repo, and hacked a little Python script that uses LLMs through the OpenAI API to keep the YAML files up-to-date semi-automatically. https://github.com/blockchain-deadlines/blockchain-deadlines.github.io/blob/main/chatgpt-updater.py Maybe it can make life a little easier for whoever decides to take over maintenance of this repo. Thanks for building this repo in the first place! :)

**Comment by ellisbrown** [comment]
pls @paperswithcode keep this updated!!! 🙏🙏🙏 
this is an essential utility as a grad student 😭

**Comment by camall3n** [comment]
Any update on this? Is there a mirror that's more up to date?

**Comment by niniack** [comment]
No afiliation with this, but one of my top search results was: 

https://nunoduarte.github.io/deadlines/

Looks reasonable at first glance. I think it might be done by @NunoDuarte

**Comment by rieck** [comment]
It’s disheartening to see this project not being actively maintained. While other deadline lists exist, none have matched the level of community support this one has received. Currently, there are 34 open pull requests with updates and new deadlines waiting to be merged!

If someone with merge permissions could step up and breathe new life into this project, it would make a huge difference. I understand this involves some effort, but I believe there are plenty of people in the community willing to help (myself included — and surely others as well). 

@joachimneu Is your AI-driven fork publicly available?

**Comment by joachimneu** [comment]
@rieck Yes, the script is described briefly [in the last paragraph of the "Contributing" section in the `README.md` here](https://github.com/blockchain-deadlines/blockchain-deadlines.github.io?tab=readme-ov-file#contributing). The subject area of the repo I maintain is different, but that shouldn't pose a big hurdle. One change would be needed: I pulled the existing `conferences.yaml` file apart into separate YAML files like [here](https://github.com/blockchain-deadlines/blockchain-deadlines.github.io/tree/main/_data/conferences_raw). The AI updater script updates these files in `_data/conferences_raw`, and [this shell script](https://github.com/blockchain-deadlines/blockchain-deadlines.github.io/blob/main/compress-conferences.sh) puts the `_raw` files back together into one `conferences.yaml` for the frontend to display. This hack was necessary because the old `conferences.yaml` doesn't capture when multiple deadlines are associated with one event (eg, multiple submission cycles for one conference, which is common in my subject area), but the model should update all deadlines for one event in one go, rather than feeding it each deadline independently (which can cause trouble, for instance if the number of deadlines changes from one year to the other). This change also makes it easier to review changes venue-by-venue before `git commit`.

**Comment by gdemelo** [comment]
Another more up-to-date mirror has been set up at:

http://aideadlines.org/

**Comment by joaopedromattos** [comment]
> Another more up-to-date mirror has been set up at:
> 
> http://aideadlines.demelo.org/

This is the closest to the original AI Deadlines. @gdemelo, I searched your repos but could not find this one. Please create a new one so we can start contributing.

Thanks for hosting!

**Comment by gdemelo** [comment]
@joaopedromattos Thanks. You can contribute to the https://github.com/ccfddl/ccf-deadlines repo, from which http://aideadlines.org/ draws new updates periodically.

**Comment by MatthewCGalbraith** [comment]
Thanks so much for the fork @gdemelo!

**Comment by Syzygianinfern0** [comment]
> [@joaopedromattos](https://github.com/joaopedromattos) Thanks. You can contribute to the https://github.com/ccfddl/ccf-deadlines repo, from which http://aideadlines.org/ draws new updates periodically.

Hi @gdemelo, I'm a bit confused by your comment that your fork draws updates from ccf-deadlines. From what I understand, the ccf-deadlines repository publishe on [ccfddl.com](https://ccfddl.com/), and it seems to include a completely different set of conferences compared to those listed on [aideadlines.org](http://aideadlines.org/). Could you please clarify what you mean?

**Comment by gdemelo** [comment]
> Hi [@gdemelo](https://github.com/gdemelo), I'm a bit confused by your comment that your fork draws updates from ccf-deadlines. From what I understand, the ccf-deadlines repository publishe on [ccfddl.com](https://ccfddl.com/), and it seems to include a completely different set of conferences compared to those listed on [aideadlines.org](http://aideadlines.org/).

@Syzygianinfern0:

The [aideadlines.org](http://aideadlines.org/) site lists AI-related deadlines only. It draws deadline information from ccf-deadlines but considers only a relevant subset.

**Comment by NielsRogge** [comment]
Hi folks, at Hugging Face we love PapersWithCode's projects. Sad it's not maintained anymore.

Hence we've created https://github.com/huggingface/ai-deadlines which is an up-to-date and maintained version of AI deadlines. It runs at https://huggingface.co/spaces/huggingface/ai-deadlines. It runs with a Github action to automatically parse new conference data from https://github.com/ccfddl/ccf-deadlines.

Feel free to give it a try and give feedback, we're happy to improve it! And open-source contributions welcome ;)

**Comment by skywalker023** [comment]
@NielsRogge Thanks a bunch, Niels! Really appreciate the effort for making this work! Happy to see the app running!

**Comment by ma-abdellaoui** [comment]
🚀 We’re excited to announce the beta launch of [SOTAPapers.com](https://sotapapers.com/)

SOTAPapers is a modern, improved alternative to Papers With Code — a platform for exploring and discovering state-of-the-art research in AI and machine learning. Like Papers With Code, it organizes and connects research papers, datasets, methods, tasks, benchmarks, and venues — but with a cleaner structure, faster search, and better filtering to help you quickly evaluate progress across domains.

Whether you’re a researcher, engineer, or academic, SOTAPapers offers a structured view of the latest advancements, making it easier than ever to stay on top of the field.

Importing the published Papers With Code database was no small task — the data was inconsistently structured and took significant effort to clean, organize, and improve. But now it’s live, and we need your help to fill in the gaps, hunt bugs, and make it even better.

You can contribute data submissions or report bugs via our GitHub: [[GitHub link here]](https://github.com/ma-abdellaoui/sotapapers-contribute)
We’re looking forward to your feedback and ideas!

**Comment by camall3n** [comment]
@ma-abdellaoui this thread is about a completely unrelated topic
